tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -714,9 +714,16 @@ def test_config_interface_speed(self, setup_config_mode, sample_intf, duthosts,
         cli_ns_option = sample_intf['cli_ns_option']
         asic_index = sample_intf['asic_index']
         duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+        # Get supported speeds for interface
+        supported_speeds = duthost.get_supported_speeds(interface)
+        # Remove native speed from supported speeds
+        if supported_speeds != None:
+            supported_speeds.remove(native_speed)
+        # Set speed to configure
+        configure_speed = supported_speeds[0] if supported_speeds else native_speed
```

Comment:
supported_speed could be an empty list after removing the native_speed from it - if the interface supports only 1 speed. Could you please add a check for that as well